src/wui/interactive_base.cc

```diff
@@ -1575,6 +1575,11 @@ void InteractiveBase::play_sound_effect(const NoteSound& note) const {
 	}
 }
 
+// request soundhandler to change music
+void InteractiveBase::change_music() {
+	g_sh->change_music();
```

Comment:
Sure, it would save two lines of code to make the call directly from the key handler function, and it's not really necessary with this one statement wrapper function. I'll commit an edit